### PR TITLE
executor: fix err check

### PR DESCRIPTION
**Pull request by Tjianke**
### What problem does this PR solve?

Issue Number: #22979  

Problem Summary:

### What is changed and how it works?

What's Changed: added err check

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

- No release note

### CODE REVIEW

**Review by lzmhhh123 — APPROVED** [review]

File: executor/aggfuncs/aggfunc_test.go [hunk 1]
```diff
@@ -508,11 +514,13 @@ func (s *testSuite) testMultiArgsMergePartialResult(c *C, p multiArgsAggTest) {
 	iter.Begin()
 	iter.Next()
 	for row := iter.Next(); row != iter.End(); row = iter.Next() {
-		partialFunc.UpdatePartialResult(s.ctx, []chunk.Row{row}, partialResult)
+		// to fix: cannot check error
```
Comment (zimulala) [inline executor/aggfuncs/aggfunc_test.go:517]: @lzmhhh123 
Please help to take a look at these `to fix` in this PR.

Comment (lzmhhh123) [reply executor/aggfuncs/aggfunc_test.go:517]: We can leave them here. And fix them in the next PR.

Comment (zimulala) [reply executor/aggfuncs/aggfunc_test.go:517]: OK.
```suggestion
		// FIXME: cannot check error
```

**Pull request merged**